contrib/opencensus-ext-azure/opencensus/ext/azure/log_exporter/__init__.py

```diff
@@ -64,7 +64,7 @@ def __init__(self, **options):
         self._worker = Worker(self._queue, self)
         self._worker.start()
         # start statsbeat on exporter instantiation
-        if not os.environ.get("APPLICATIONINSIGHTS_STATSBEAT_DISABLED_ALL"):
+        if os.environ.get("APPLICATIONINSIGHTS_STATSBEAT_DISABLED_ALL") != "True":
```

Comment:
APPLICATIONINSIGHTS_STATSBEAT_DISABLED_ALL=true will not work in this case, we usually just check for the env variable to exist and value is not that important, if you want to check for value ensure lower case and extra spaces doesn't mess with the check

Reply:
If it's normal practice to check if it exists then that is fine with me. Will leave it as a `None` check then.

Reply:
value matters though.. 

convert string to boolean and then check it.  Boolean.parseBoolean(value) (java way) handle case for you.

Reply:
I'm fine either way, just trying to be consistent with node js sdk. @hectorhdzg  thoughts?

Reply:
Like I mentioned node.js SDK does this kind of only checking for env to exist and not checking the value, we also mention this in docs when apply, is up to you how to handle it, casting works as well

Reply:
Will leave it as None.